addons/stock/stock.py

```diff
@@ -2861,23 +2906,6 @@ def _resolve_inventory_line(self, cr, uid, inventory_line, context=None):
         return stock_move_obj.create(cr, uid, vals, context=context)
 
     def restrict_change(self, cr, uid, ids, theoretical_qty, context=None):
```

Comment:
add a comment on both `restrict_change` and `on_change_product_id` to indicate they should be deleted in saas-6/master